bootstrap.d/dev-libs.y4.yml

```diff
@@ -1174,8 +1174,8 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://github.com/thom311/libnl.git'
-      tag: 'libnl3_9_0'
-      version: '3.9.0'
+      tag: 'libnl3_11_0'
+      version: '3.11.0'
```

Comment:
This needs a revision reset. Please also test the patches we now shipped in the meantime.